lib/Internal/Placeholder.php

```diff
@@ -3,7 +3,8 @@
 namespace Amp\Internal;
 
 use Amp\Failure;
-use Interop\Async\{ Loop, Promise };
+use Interop\Async\Loop;
```

Comment:
Both, IDE changed it to

```
use Interop\Async\{
    Loop, Promise
};
```

I changed it then to avoid those stupid group uses.